facade-app/package.json

```diff
@@ -44,6 +44,14 @@
       "last 1 safari version"
     ]
   },
+  "jest": {
+    "collectCoverageFrom": [
```

Comment:
Can this config be part of the jest.config.ts file instead of package.json?

Reply:
I tried jest.config.ts but these properties are not working in that file hence had to move it to package.json.